Bottom: 342c823e9bb776cc82b1619068f5a40fb6dfaade
Top:    32110385b445e4c6368bc1b0a9f9c9f9a9374d21
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:58:53 +0100

Fix for PCB+GL Experimental


---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 5bfbffd..124c00e 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -853,9 +853,9 @@ do_hole (const BoxType *b, void *cl)
 }
 
 static bool
-polygon_contains_user_holes (PolygonType *polygon)
+pour_contains_user_holes (PourType *pour)
 {
-  return (polygon->HoleIndexN > 0);
+  return (pour->HoleIndexN > 0);
 }
 
 
@@ -889,7 +889,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box /*, bool forc
   /* Polygon has holes.. does it have any user-drawn holes?
    * If so, it must be masked with a _new_ stencil bit.
    */
-  use_new_stencil = force_new_stencil | polygon_contains_user_holes (poly);
+  use_new_stencil = force_new_stencil || true; // | polygon_contains_user_holes (poly);
 
   if (use_new_stencil)
     {
